cloudbuild.yaml

```diff
@@ -28,9 +28,10 @@ steps:
       # appropriately for the image before running make
       /buildx-entrypoint version
 
+      REALTAG=$(git describe --tags --always --dirty --match 'v1.*')
```

Comment:
the problem currently is that `$_SHORT_TAG` is coming from command `git describe --tags --always --dirty`. However, if we have multiple tags in same commit, like v1.27.0 does:

```
 cloud-provider-openstack [master] % git checkout v1.27.0
Note: switching to 'v1.27.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 309df4c7 Merge pull request #2203 from zetaab/bumpbversions
 cloud-provider-openstack [tags/openstack-cinder-csi-2.27.0] % git describe --tags --always --dirty
openstack-cinder-csi-2.27.0
 cloud-provider-openstack [tags/openstack-cinder-csi-2.27.0] % git describe --tags --always --dirty --match 'v1.*'
v1.27.0
```

So it should fix the issue

Reply:
another example: if we run in current master branch:

```
 cloud-provider-openstack [master] % git describe --tags --always --dirty 
openstack-cinder-csi-2.28.0-alpha.1
 cloud-provider-openstack [master] % git describe --tags --always --dirty --match 'v1.*'
v1.27.0-alpha.0-22-ge8d89e58
```

so it looks like much better with `--match 'v1.*'` flag